src/lib/y2users/users_simple/writer.rb

```diff
@@ -23,7 +23,7 @@
 
 module Y2Users
   module UsersSimple
-    # Class for writing users configuration into the old UsersSimple Yast Module
+    # Class for writing users config into the old UsersSimple Yast Module
```

Comment:
I am not sure if we should use acronym also in comment. That is why I keep in comments full name.

Reply:
Ok, I can revert it. I replaced it automatically when fixing a conflict.